src/apps/concepts/pages/ViewConceptsPage.tsx

```diff
@@ -192,8 +197,9 @@ const ViewConceptsPage: React.FC<Props> = ({
       sortBy: sortBy,
       dataTypeFilters: initialDataTypeFilters,
       classFilters: initialClassFilters,
+      generalFilters: initialGeneralFilters,
       sourceFilters: initialSourceFilters,
-      includeRetired: true,
+      includeRetired: false,
```

Comment:
hi @jwnasambu just confirming does this `includeRetired` property is working fine now? as I am not able to see `concepts without the Retired concepts` even after  `includeRetired: false,`

Reply:
I think we can make includeRetired to depend on the generalFilters and the status of the `includeRetired` checkbox, that way, we send the right request.
` initialGeneralFilters.includes('IncludeRetired') ? Number(1) : Number(0)`

Reply:
I agree with @hadijahkyampeire on this point, but as JavaScript has a built-in boolean type, there's really no need to convert to numbers or anything.

```ts
includeRetired: initialGeneralFilters.includes('IncludeRetired'),
```